### PR TITLE
Eng 2284 Integrate MariaDB to Integration Tests

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
 - [x] I ran the tests for MariaDB and they all pass (on Linux EC2 instance).
 - [x] I added instructions on creating a MariaDB database for testing [here](https://www.notion.so/aqueducthq/Data-Integration-Setup-Guides-d1fad9f3c8c0467b841d44058f95f879)
 - [x] Updated `test-config.yml` on AWS

Cannot connect to the MariaDB or MySQL Docker container on M1 Macs. Issue filed here: https://linear.app/aqueducthq/issue/ENG-2422/mac-m1-cannot-connect-to-mysqlmariadb 

I have an instance up to test. Will delete instance once this PR is merged.
 
## Related issue number (if any)
ENG 2284

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [N/A] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [N/A] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: integration_tests/sdk/setup_integration.py [hunk 1]
```diff
@@ -134,8 +134,13 @@ def _setup_mongo_db_data(client: Client, mongo_db: MongoDBIntegration) -> None:
 
 def _setup_relational_data(client: Client, db: RelationalDBIntegration) -> None:
     # Find all the tables that already exist.
-    existing_table_names = set(db.list_tables()["tablename"])
-
+    columns = list(db.list_tables().columns)
+    if "tablename" in columns:
+        existing_table_names = set(db.list_tables()["tablename"])
+    elif "table_name" in columns:
```
Comment (kenxu95) [inline integration_tests/sdk/setup_integration.py:140]: How hard would it be to edit the mariaDB list_tables() to return the tables in a "tablename" column? I'd much rather make that api consistent then branch in our testing logic.

File: integration_tests/sdk/test-credentials-example.yml [hunk 2]
```diff
@@ -3,10 +3,35 @@ apikey: <APIKEY>
 # Each integration entry starts with the name of the integration, followed
 # by the list of credentials needed to connect to it.
 data:
+  test_athena:
+    type: Athena
+    database: test
+    region: us-west-1
+    # We should all have access to this bucket, so no need to change the value here.
+    output_location: s3://aqueduct-athena-temp/tmp/
```
Comment (kenxu95) [inline integration_tests/sdk/test-credentials-example.yml:11]: Will this Athena thing go away with a rebase?

**Review by kenxu95 — CHANGES_REQUESTED** [review]

File: integration_tests/sdk/test-credentials-example.yml (hunk 2 above)
Comment (eunice-chan) [reply integration_tests/sdk/test-credentials-example.yml:11]: This is what is currently on main: https://github.com/aqueducthq/aqueduct/blob/6b1319234fd4c3cfae427f28e4468207e54c8772/integration_tests/sdk/test-credentials-example.yml#L44
Should I remove it?

File: integration_tests/sdk/setup_integration.py (hunk 1 above)
Comment (eunice-chan) [reply integration_tests/sdk/setup_integration.py]: Easier than expected! Updated.

**Review by kenxu95 — APPROVED** [review]

**Pull request merged**